### ISSUE TITLE
Depreciation Warning

### ISSUE THREAD

**Issue by opencityint** [zonak/grunt-ftp-deploy#117]
Uploading from command line
grunt ftp
success! but warnings which I don't remember from a while back(?) .ftppass and inline credentials all standard with double quotes.
-------
(node:78761) DeprecationWarning: Ftp.raw[user](args): Use Ftp.raw('user args') instead.
(node:78761) DeprecationWarning: Ftp.raw[pass](args): Use Ftp.raw('pass args') instead.
(node:78761) DeprecationWarning: Ftp.raw[type](args): Use Ftp.raw('type args') instead.
(node:78761) DeprecationWarning: Ftp.raw[cwd](args): Use Ftp.raw('cwd args') instead.
(node:78761) DeprecationWarning: Ftp.raw[quit](args): Use Ftp.raw('quit args') instead.
-------

thanks in advance

**Comment by yepes** [comment]
Getting the same messages here

**Comment by maelfrize** [comment]
Same here

Thanks for the great library 👍

**Comment by Kelderic** [comment]
This is still happening. It has been fixed upstream.

**Comment by spoji** [comment]
Adding to the comment of @Kelderic , I've created a PR that fix all the warnings for me. You can take a look at it and change the files locally while we are waiting for @zonak to accept and merge the PR..

Cheers!

**Comment by Kelderic** [comment]
Unfortunately, I'm pretty sure that this project has been abandoned, @spoji.

**Comment by spoji** [comment]
Yeah... Last updates were 3 years ago. The pull request should at least help others having the same issue and need a solution to it.

**Comment by spoji** [comment]
I stand corrected. The PR has been merged back and npm has been updated to 0.2.0 which include the fix.

https://www.npmjs.com/package/grunt-ftp-deploy

@zonak This issue can be closed.
